test/Visualization_test.jl

```diff
@@ -0,0 +1,32 @@
+@testset "Visualization" begin
+    using CairoMakie
+    using Tenet
+    using NetworkLayout: Spring
+
+    @testset "`plot``" begin
+        tn = TensorNetwork([
+            Tensor(rand([2, 2, 2, 2]...), tuple([:x, :y, :z, :t]...)),
+            Tensor(rand([2, 2]...), tuple([:x, :y]...)),
+            Tensor(rand([2]...), tuple([:x]...))
+            ])
```

Comment:
```suggestion
        tn = TensorNetwork([
            Tensor(rand(2, 2, 2, 2), (:x, :y, :z, :t)),
            Tensor(rand(2, 2), (:x, :y)),
            Tensor(rand(2), (:x))
            ])
```

Reply:
This block of code gives error, maybe we have to look at this since it only fails when the `labels` of the tensor with a single dimension is a `tuple. For example this fails:
```julia
tn = TensorNetwork([
           Tensor(rand(2, 2, 2, 2), [:x, :y, :z, :t]),
           Tensor(rand(2, 2), [:x, :y]),
           Tensor(rand(2),(:x))
           ])
```
But this doesn't:
```julia
tn = TensorNetwork([
           Tensor(rand(2, 2, 2, 2), (:x, :y, :z, :t)),
           Tensor(rand(2, 2), (:x, :y)),
           Tensor(rand(2),[:x])
           ])
```
For now, I will keep the labels in a `Vector`.

Reply:
Ah yes, it’s because a single element tuple needs a coma so `(:x,)`